core/src/labels/labels.cpp

```diff
@@ -313,6 +313,8 @@ const std::vector<TouchItem>& Labels::getFeaturesAtPoint(const View& _view, floa
 
             for (auto& label : labelMesh->getLabels()) {
 
+                if (label->state() != Label::State::visible) { continue; }
```

Comment:
You can directly use https://github.com/tangrams/tangram-es/blob/master/core/src/labels/label.cpp#L169